packages/site/src/components/App.js

```diff
@@ -58,17 +56,18 @@ const Root = createStyledComponent('div', styles.app);
 const Nav = createStyledComponent(_Nav, styles.nav);
 const Main = createStyledComponent('main', styles.main);
 
-export default function App({ className }: Props) {
+export default function App({ className, demos }: Props) {
+  if (demos.length === 1) {
```

Comment:
Nice. Of note, the site currently displays multiple components (demos) per page to get around the lack of routing.